# HG changeset patch
# User Brian <runningnak3d@gmail.com>
# Date 1277924280 21600
# Branch trunk
# Node ID 61015aa420a7ff06f635b2782966cf1181a7384c
# Parent  98590f19d2b046fea22be18d6d0e0e412fb621ca
* Update / Add some opcodes
* Thanks TOM_RUS

diff -r 98590f19d2b0 -r 61015aa420a7 src/server/game/Server/Protocol/Opcodes.h
--- a/src/server/game/Server/Protocol/Opcodes.h	Wed Jun 30 12:46:19 2010 -0600
+++ b/src/server/game/Server/Protocol/Opcodes.h	Wed Jun 30 12:58:00 2010 -0600
@@ -1343,7 +1343,11 @@
     SMSG_UNKNOWN_1304                               = 0x518, // something with player movement (move event 58?), speed packet
     UMSG_UNKNOWN_1305                               = 0x519, // not found
     UMSG_UNKNOWN_1306                               = 0x51A, // not found
-    NUM_MSG_TYPES                                   = 0x51B
+    CMSG_COMMENTATOR_SKIRMISH_QUEUE_COMMAND         = 0x51B, // lua: CommentatorSetSkirmishMatchmakingMode/CommentatorRequestSkirmishQueueData/CommentatorRequestSkirmishMode/CommentatorStartSkirmishMatch
+    SMSG_UNKNOWN_1308                               = 0x51C, // event EVENT_COMMENTATOR_SKIRMISH_QUEUE_REQUEST, CGCommentator::QueueNode
+    SMSG_UNKNOWN_1309                               = 0x51D, // event EVENT_COMMENTATOR_SKIRMISH_QUEUE_REQUEST
+    SMSG_UNKNOWN_1310                               = 0x51E, // some compressed packet?
+    NUM_MSG_TYPES                                   = 0x51F
 };
 
 /// Player state
